table/sst_file_writer.cc

```diff
@@ -135,25 +133,30 @@ struct SstFileWriter::Rep {
     file_info.num_range_del_entries++;
     file_info.file_size = builder->FileSize();
 
-    InvalidatePageCache(false /* closing */);
-
-    return Status::OK();
+    return InvalidatePageCache(false /* closing */);
   }
 
-  void InvalidatePageCache(bool closing) {
+  Status InvalidatePageCache(bool closing) {
+    Status s;
```

Comment:
Please initialize s = Status::OK(); for clarity

Reply:
Done.